econml/ortho_forest.py

```diff
@@ -330,7 +374,26 @@ def _get_weights(self, X_single):
                 w2[unique] += leaf_weight * counts
             else:
                 w2[weight_indexes] += leaf_weight
-        return (w1 / self.n_trees, w2 / self.n_trees)
+        return (w1 / len(tree_range), w2 / len(tree_range))
+
+    def _get_blb_indices(self, X):
+        """Get  data indices for every tree under the little bags split."""
+        # Define subsample size
+        subsample_size = X.shape[0] // 2
+        if not self.bootstrap:
+            if self.subsample_ratio > 1.0:
+                # Safety check
+                self.subsample_ratio = 1.0
```

Comment:
Does this merit a warning, or could it happen for reasons that a user can't control?

Reply:
Added a warning to reflect this change.